pupil-api/src/App.ts

```diff
@@ -12,6 +12,8 @@ import * as winston from 'winston'
 import * as expressWinston from 'express-winston'
 import * as azure from './azure'
 const config = require('./config')
+const corsOptions = require('../../admin/helpers/cors-options')
```

Comment:
how do we set the permitted domains for CORS rules?

Reply:
I am not sure that we should rely on this behaviour - that it requires a file from another project, in this case, `admin`.    This file also includes the admin `config` file which seems potentially dangerous.

It makes packaging less obvious.  Let's not require files outside of the current application's directory.

Reply:
@jon-shipley agreed